src/mutable_data.rs

```diff
@@ -8,13 +8,15 @@
 // Software.
 
 use crate::errors::{EntryError, Error};
+use crate::request::{Request, Requester};
+use crate::PublicKey;
 use crate::XorName;
+use maidsafe_utilities::serialisation::serialise;
```

Comment:
We should use `bincode::serialize` instead, so `maidsafe_utilities` dependency is not required.